data.js

```diff
@@ -589,6 +589,14 @@ module.exports = [
     url: "https://github.com/alexanderGugel/micro-events",
     source: "https://raw.githubusercontent.com/alexanderGugel/micro-events/master/index.js"
   },
+  {
+    name: "svelte-inline-style",
+    github: "rozek/svelte-inline-style",
+    tags: ["svelte3", "inline-styles", "svelte-action"],
+    description: "Efficiently apply inline styles to Svelte components.",
+    url: "https://github.com/rozek/svelte-inline-style",
+    source: "https://raw.githubusercontent.com/rozek/svelte-inline-style/main/src/svelte-inline-style.ts"
```

Comment:
From the readme - emphasis on "ready-to-use", which means you should not link to a TypeScript source file:

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified (see below)